src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Entity;
+
+interface ActivatableToggle
```

Comment:
well actually this interface is solely concerned with whether something is active or not, setting it could happen anywhere